http_check/datadog_checks/http_check/http_check.py

```diff
@@ -38,43 +35,52 @@ class HTTPCheck(NetworkCheck):
     SC_STATUS = 'http.can_connect'
     SC_SSL_CERT = 'http.ssl_cert'
 
-    def __init__(self, name, init_config, agentConfig, instances=None):
-        NetworkCheck.__init__(self, name, init_config, agentConfig, instances)
+    HTTP_CONFIG_REMAPPER = {
+        'client_cert': {'name': 'tls_cert'},
+        'client_key': {'name': 'tls_private_key'},
+        'disable_ssl_validation': {'name': 'tls_verify', 'invert': True, 'default': True},
+        'ignore_ssl_warning': {'name': 'tls_ignore_warning'},
+        'ca_certs': {'name': 'tls_ca_cert'},
+    }
+
+    def __init__(self, name, init_config, instances):
+        super(HTTPCheck, self).__init__(name, init_config, instances)
 
         self.ca_certs = init_config.get('ca_certs')
         if not self.ca_certs:
             self.ca_certs = get_ca_certs_path()
 
+        self.HTTP_CONFIG_REMAPPER['ca_certs']['default'] = self.ca_certs
+
+        if is_affirmative(self.instance.get('disable_ssl_validation', True)):
```

Comment:
doesnt the mapping to `tls_verify` already do that?

Reply:
The wrapper logic sets `verify` to the `tls_ca_cert` path if it exists. But the `http_check` overrides `verify` if the user has `disable_ssl_validation` set to True. If there is a `tls_ca_cert` set, the wrapper does not check the value of `tls_verify`.